src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Python/Tov.cpp

```diff
@@ -0,0 +1,29 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <boost/python.hpp>
+#include <string>
+
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Tov.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/PolytropicFluid.hpp"
+
+namespace bp = boost::python;
+
+namespace gr {
+namespace Solutions {
+namespace py_bindings {
+
+void bind_tov() {
+  bp::class_<TovSolution, boost::noncopyable>(
+      "Tov",
+      // Only allowing a polytropic EOS here for now since boost::python has
```

Comment:
I haven't tried using virtual base classes with bp. What troubles have you run into? Are they so bad we should consider switching to bypind11 in the future?

Edit:
There's some info  here: https://www.boost.org/doc/libs/1_61_0/libs/python/doc/html/tutorial/tutorial/exposing.html#tutorial.exposing.inheritance

Reply:
I got virtual base classes working now, details below. There might be another reason to switch to pybind11 though: Looks like it deals with numpy interoperability quite well, e.g. provides functionality to vectorize a C++ function that takes and returns `double`s into a Python function that takes and returns Numpy arrays.

Re virtual base classes: The tutorial you mentioned is the one I followed to get the virtual base classes working before. The issue I ran into turned out to be only tangentially related to virtual base classes. Apparently the type of the `equation_of_state` argument must be correctly specified as const-reference when it's a virtual base class, but doesn't need to be when it's a non-virtual subclass. Mostly I find that bp compiler errors are quite unhelpful, but it seems to be working OK.